### PR TITLE
Fix coverity scan problem

### PR DESCRIPTION
**Pull request by rhatdan**
Coverity is complaining about a leaded file descriptor, since
we never close dummyfd.

This change will close the dummyfd when the program exits.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe PTAL

File: conmon/conmon.c [hunk 1]
```diff
@@ -869,10 +869,10 @@ static void setup_terminal_control_fifo()
 	int dummyfd = open(ctl_fifo_path, O_WRONLY | O_CLOEXEC);
```
Comment (haircommander) [inline conmon/conmon.c]: why not just set this as _cleanup_close_ ? Does it need to be in the scope of main to work?

Comment (rhatdan) [reply conmon/conmon.c]: I believe so.
I will let Giuseppe judge, I think this has to be open for the entire run of conmon and only closed when it exits, but coverity sees this as a leak.

Comment (giuseppe) [reply conmon/conmon.c]: yes, it needs to be in the scope of main, otherwise it will be closed as soon as we leave `setup_terminal_control_fifo`

**Comment by rhatdan** [comment]
@haircommander @umohnani8 @mrunalp PTAL

**Pull request merged**